addons/mass_mailing_sms/models/mailing_mailing.py

```diff
@@ -136,6 +152,14 @@ def _action_view_traces_filtered(self, view_filter):
                                (self.env.ref('mass_mailing_sms.mailing_trace_view_form_sms').id, 'form')]
         return action
 
+    def _action_view_documents_filtered_get_traces(self, view_filter):
```

Comment:
Would move to 'mass_mailing', like we support delivered in 'mail' to simplify code. Would allow to remove this method and keep the filtering in '_action_view_documents_filtered' and avoid yet another sub-method call and override ^^